server/development.ini

```diff
@@ -16,6 +16,12 @@ pyramid.includes =
 
```

Comment:
Please remove `development.ini` from git. But correct way was using `git mv development.ini development.ini.sample` and git would know that you are renaming file.